lib/ansible/modules/remote_management/ucs/ucs_timezone.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_timezone
+
+short_description: Configures timezone on Cisco UCS Manager
+
+version_added: "2.7"
+
+description:
+    - Configures timezone on Cisco UCS Manager.
+    - Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+
+options:
+    state:
+        description:
+            - If C(present), will set or update timezone.
+            - If C(absent), will unset timezone.
+        choices: [present, absent]
+        default: present
+
+    admin_state:
+        description: admin state indicates if the timezone confguration is enabled and being utilized by UCS Manager
+            or disabled and being ignored by UCS Manager
+        choices: ['disabled', 'enabled']
+        default: "enabled"
+
+    timezone:
+        description:
+            - The timezone name. Time zone names are from the tz database U(https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
```

Comment:
So for the documentation the first line usually is the description of the parameter, subsequent lines add more detailed information to values. So I would split it up like this:
```yaml
  timezone:
    description:
    - The timezone name.
    - Time zone names are from the L(TZ database,https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
    - The timezone name is case sensitive.
...
```

Reply:
changes made